services/src/main/java/org/exoplatform/news/NewsUtils.java

```diff
@@ -76,4 +81,12 @@ private static Identity loadUser(String username) {
     return identityManager.getOrCreateIdentity(OrganizationIdentityProvider.NAME, username);
   }
 
+  public static List<Space> getSpacesWhichIsManagerOrRedactor(String userId) throws SpaceException {
```

Comment:
```suggestion
  public static List<Space> getRedactorOrManagerSpaces(String userId) throws SpaceException {
```
SpaceException to be checked